listings/ch01-applications/crowd_fund/Scarb.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "crowd_fund"
+version = "0.1.0"
+
+# See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
+
+[dependencies]
+starknet = ">=2.3.1"
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.12.0" }
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.8.0-beta.0" }
```

Comment:
Can you add these dependencies in the root workspace and add them here:
```toml
snforge_std.workspace = true
openzeppelin.workspace = true
```

As explained here: https://github.com/NethermindEth/StarknetByExample/blob/main/CONTRIBUTING.md#adding-a-new-cairo-program

Reply:
I am facing some issue here, I have pushed the code for you to review it once. I am getting:

```
Caused by:
    0: error inheriting `openzeppelin` from workspace root manifest's `workspace.dependencies.openzeppelin`
    1: dependency `openzeppelin` not found in workspace
```